crates/re_viewer_context/src/blueprint_helpers.rs

```diff
@@ -61,4 +72,12 @@ impl ViewerContext<'_> {
             }
         }
     }
+
+    pub fn choose_store_for_query(&self, query: &re_data_store::LatestAtQuery) -> &DataStore {
+        if query.timeline == blueprint_timeline() {
```

Comment:
This feels dangerous. What happens if a user has a timeline called "blueprint"? Is this now a reserved name for a timeline?

If we're going down this route, I suggest we call the timeline `rerun.blueprint`

Reply:
Yeah, that's a good point. As annoying as it is, maybe it's worth just plumbing everything through explicitly.